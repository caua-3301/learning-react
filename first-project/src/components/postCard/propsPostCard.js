export const props = {
  photoUrl: 'imagens/foto1.jpeg',
  completed: true,
  title: 'titulo de teste vindo da prop PostCard',
};
