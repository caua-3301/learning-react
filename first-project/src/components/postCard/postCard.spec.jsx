import { screen, render } from "@testing-library/react";
import { props } from "./propsPostCard";
import { PostCard } from ".";

describe("<PostCard />", () => {
  it("should show itens", () => {
    render(<PostCard {...props} />);

    //debug();

    expect(screen.getByRole('img')).toBeInTheDocument();
  })

  it("should make a snapshot", () => {
    render(<PostCard {...props} />);

    //debug();

    expect(document.body).toMatchSnapshot();
  })
});
