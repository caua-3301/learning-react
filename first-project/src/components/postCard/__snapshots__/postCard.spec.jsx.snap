// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostCard /> should make a snapshot 1`] = `
<body>
  <div>
    <div
      class="card"
    >
      <img
        src="imagens/foto1.jpeg"
      />
      <h4>
        Titulo: 
        titulo de teste vindo da prop PostCard
         
      </h4>
      <p>
         
        Conpletada
         
      </p>
    </div>
  </div>
</body>
`;
